### ISSUE TITLE
[ts] Test Chrome Browser Error - Cannot read property 'VERTEX_SHADER' of null

### ISSUE THREAD

**Issue by BlytzStyle** [EsotericSoftware/spine-runtimes#862]
I try to put in my application in a web page animation but is not compatible with Chrome browser?

Chrome  run test error
Uncaught TypeError: Cannot read property 'VERTEX_SHADER' of null
    at Shader.compile (Shader.ts:59)
    at new Shader (Shader.ts:52)
    at Function.Shader.newColoredTextured (Shader.ts:204)
    at new SpineWidget (Widget.ts:72)
    at Function.SpineWidget.loadWidget (Widget.ts:308)
    at Function.SpineWidget.loadWidgets (Widget.ts:288)
    at HTMLDocument.SpineWidget.ready (Widget.ts:317)

Uncaught TypeError: Cannot read property 'VERTEX_SHADER' of null
    at Shader.compile (spine-webgl.js:6677)
    at new Shader (spine-webgl.js:6669)
    at Function.Shader.newColoredTextured (spine-webgl.js:6779)

Safari run test ok 
Firefox run test ok

A well designed application must be compatible with any browser! Right?

You can rectify this problem? Thank you!

**Comment by badlogic** [comment]
The widget does indeed work in the browser. How do you include the widget code (`spine-widget.js`) in your web app?

**Comment by BlytzStyle** [comment]
I try to explain what I do when something goes wrong.

Running your applications (samples) to understand whether libraries are well configured.

take your example spine-runtime, put it in the root (localhost) and do a test:

  http: // localhost / spine-runtimes / spine-ts / widget / example /

Your code is an example of nothing more or less

and I result from the attached picture.
![2017-03-22 01 28 02 pm](https://cloud.githubusercontent.com/assets/21214416/24197810/704e6ad6-0f03-11e7-85f0-644f6e0b6cdc.png)

google chrome vers. : 57.0.0287.110 (64bit)

Thank you !

**Comment by BlytzStyle** [comment]
I downloaded Chrome again,
I tried this result does not change...
![2017-03-22 04 33 27 pm](https://cloud.githubusercontent.com/assets/21214416/24206517/bc167bbe-0f1e-11e7-9313-94c49a50e804.png)

second problem (version master)
widget/example/index.html
var animIndex = 0;
		widget.canvas.onclick = function () {
			animIndex++;
            **-          let animations = widget.skeleton.data.animations;
           +          var animations = widget.skeleton.data.animations;**
			if (animIndex >= animations.length) animIndex = 0;
			widget.setAnimation(animations[animIndex].name);
		}

osx safari with this change ok
osx chrome error

master version iPhone 5s wifi load localhost http://192.168.xxx.xxx/spine-runtime-master/spine-ts/widget/example test (safari and chrome ) loading page animation begins to play after that blocks and reload the page.

Thanks for understanding.

**Comment by badlogic** [comment]
Ok, I'm running Chrome version Version 56.0.2924.87 (64-bit) as shown above, and can't reproduce the issue. I then updated to Chrome Version 57.0.2987.110 (64-bit) and it works as well. Frankly, I'm puzzled. I also tried in the latest Safari for macOS (Version 10.0 (11602.1.50.0.10) and Firefox (52.0.1 (64-bit)), all on macOS. I also tried on Android Chrome (Version 56.0.2924.87). I currently do not have an iOS device to test Safari mobile.

I've fixed the issue in `index.html`.

Do you have any extensions loaded in your desktop Chrome installation that may mess with the JavaScript?

**Comment by BlytzStyle** [comment]
**solved**: chrome flag start up

chrome: // flags

Override software rendering list Mac, Windows, Linux, Chrome OS, Android
Overrides the built-in software rendering list and Enables GPU-acceleration on unsupported system configurations. # Ignore-gpu-blacklist
**Disable**

Thank you

**Comment by badlogic** [comment]
Thanks for getting back to me, didn't occur to me that hardware rendering might be disabled if it worked in Firefox and Safari for you.

**Comment by BlytzStyle** [comment]
iPhone 5 (Safari browser) break run page and reload page when debug is true.

**Comment by badlogic** [comment]
Sorry, I don't understand. Can you rephrase?

**Comment by BlytzStyle** [comment]
I tried to run your example on a iphone 5s with Safari browser, I noticed that after 2 seconds of animated page locks and recharge, we have disabled the debug object "debug: false"
                  ......
new spine.SpineWidget("spine-widget-inline", {
	jsonContent: json,
	atlasContent: atlas,
	imagesPath: "assets/",
	animation: "animation",
	backgroundColor: "#00000000",
	**debug: false**,
	success: function (widget) {
		.......
 and this does not happen. I think you as the animation is presented to the debug.

**Comment by badlogic** [comment]
Thanks, I'll try to reproduce this!
